v20.2/liquibase.md

```diff
@@ -6,22 +6,20 @@ toc: true
 
 This page walks you through a series of simple database schema changes using the [Liquibase](https://www.liquibase.org/get-started/how-liquibase-works) command-line tool and the [CockroachDB SQL shell](cockroach-sql.html).
 
-We have tested Liquibase enough to claim **beta-level** support.
-
 For detailed information about using Liquibase, see the [Liquibase documentation site](https://docs.liquibase.com/home.html).
 
 ## Before you begin
 
 Before you begin the tutorial, do the following:
 
 1. [Install CockroachDB](install-cockroachdb.html), and [start a secure cluster](secure-a-cluster.html). When starting your cluster, make sure that you generate cluster certificates, create the `bank` database, and create the `max` user.
-1. Download and install a Java Development Kit. Liquibase supports Java 8+. In this tutorial, we use [AdoptOpenJDK](https://adoptopenjdk.net/) 11.
+1. Download and install a Java Development Kit. Liquibase supports Java 8+. In this tutorial, we use [AdoptOpenJDK](https://adoptopenjdk.net/) 8.
```

Comment:
Is there a reason we lowered to JDK 8?

Reply:
JDK 8 is much more popular than JDK 11, so it's likely than developers looking at our docs will be using JDK 8. Also, we use JDK 8 in both of our Spring tutorials. So it would be good to be consistent across our docs tutorials. Also, Liquibase docs point to a JDK 8 installation: https://docs.liquibase.com/concepts/installation/installation-linux-unix-mac.html.

So I think JDK 8 would be a good move for this tutorial. But I'm not super familiar with the Java community. @ianjevans and/or @kai-niemi might have opinions on the topic as well, seeing as they have both have a lot of experience with the Java community.

Reply:
I think recommending "JDK 1.8 (or later)" for demo's is best for both consistency and compability. JDK 8 is still widely adopted and offers better compability with the rest of the community frameworks/libraries (like Spring, Hibernate, Liquibase etc) for years to come. A lot of things happened after JDK 8 both in terms of the platform and also Oracle's re-licensing. While both the language and platform features have evolved significantly since JDK8, none of the demos depend on that (little reason to). Hence you could use JDK11+ and still run all demos but its not mandatory.